helper/helper-common/src/main/java/com/linkedin/avroutil1/compatibility/SchemaComparisonConfiguration.java

```diff
@@ -15,16 +15,16 @@ public class SchemaComparisonConfiguration {
    * behaves like avro &lt;= 1.7.2 - non-string props on fields or types are ignored
    */
   public static final SchemaComparisonConfiguration PRE_1_7_3 = new SchemaComparisonConfiguration(
-      true, false, false, false, true, false,  Collections.emptySet()
+      true, false, false, false, true, false,  false, Collections.emptySet()
   );
   /**
    * same as {@link #STRICT} but allows int default values to match (round) float default values
    */
   public static final SchemaComparisonConfiguration LOOSE_NUMERICS = new SchemaComparisonConfiguration(
-      true, true, true, true, true, true, Collections.emptySet()
+      true, true, true, true, true, true, false, Collections.emptySet()
   );
   public static final SchemaComparisonConfiguration STRICT = new SchemaComparisonConfiguration(
-      true, true, true, false, true, true,  Collections.emptySet()
+      true, true, true, false, true, true,  true, Collections.emptySet()
```

Comment:
This would start failing existing users of this configuration.. Lets create a separate config with this check turned on.. something like `STRICT_COMPARE_FIELD_DOCS` ?